main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"dynamic-buildkite-template/generator"
+	"os"
+)
+
+func main() {
+	var trivyPlugin = "v1.18.0"
```

Comment:
This shouldn't be hardcoded instead should be passed in params

Reply:
fixed